<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>disease_description</class>
 <widget class="QWidget" name="disease_description">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Значения признаков при заболевании</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="currentText">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>281</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>110</y>
     <width>341</width>
     <height>231</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::TabFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTipDuration">
    <number>0</number>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="elideMode">
    <enum>Qt::ElideNone</enum>
   </property>
   <property name="tabBarAutoHide">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="tab_log">
    <attribute name="title">
     <string>Логический</string>
    </attribute>
    <widget class="QRadioButton" name="radioButtonTrue">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>True</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonFalse">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>False</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_num">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Количественный</string>
    </attribute>
    <widget class="QLineEdit" name="lineEditMin">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditMax">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:white;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>min</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>max</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_num_poss">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Возможные значения</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_qual">
    <attribute name="title">
     <string>Качественный</string>
    </attribute>
    <widget class="QListWidget" name="listWidgetQual">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>211</width>
       <height>131</height>
      </rect>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>321</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Выберите одно или несколько значений из списка</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonNext">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonBack">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <zorder>comboBox</zorder>
  <zorder>tabWidget</zorder>
  <zorder>pushButtonNext</zorder>
  <zorder>pushButtonBack</zorder>
  <zorder>listWidget</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
